src/webgpu/shader/validation/decl/util.ts

```diff
@@ -40,3 +48,128 @@ fn ${arg.name}() {
 }`;
   }
 }
+
+/**
+ * @returns a WGSL var declaration with given parameters for variable 'x' and
+ * store type i32.
+ */
+export function declareVarX(addressSpace: AddressSpace | '', accessMode: AccessMode | ''): string {
+  const parts: string[] = [];
+  if (addressSpace && kAddressSpaceInfo[addressSpace].binding) parts.push('@group(0) @binding(0) ');
+  parts.push('var');
+
+  const template_parts: string[] = [];
+  if (addressSpace) template_parts.push(addressSpace as string);
```

Comment:
I think you can drop the `as string` here

Reply:
Fixed.